handsontable/src/plugins/contextMenu/menu.js

```diff
@@ -432,10 +447,20 @@ class Menu {
       } else {
         this.setPositionBelowCursor(cursor);
       }
-      if (cursor.fitsOnRight(this.container)) {
-        this.setPositionOnRightOfCursor(cursor);
-      } else {
-        this.setPositionOnLeftOfCursor(cursor);
+
+      if (this.hot.isLtr()) { // ltr mode
+        if (cursor.fitsOnRight(this.container)) {
+          this.setPositionOnRightOfCursor(cursor);
+        } else {
+          this.setPositionOnLeftOfCursor(cursor);
+        }
+      } else { // rtl mode
+        // eslint-disable-next-line no-lonely-if
```

Comment:
Probably it's not needed anymore. Please lint code after removing.

```suggestion
```

Reply:
I added it because of eslint error, and I want to keep this formatting like now. It is much more clear than:

```js
if (this.hot.isLtr()) { // ltr mode
  if (cursor.fitsOnRight(this.container)) {
    this.setPositionOnRightOfCursor(cursor);
  } else {
    this.setPositionOnLeftOfCursor(cursor);
  }
} else if (cursor.fitsOnLeft(this.container)) {
  this.setPositionOnLeftOfCursor(cursor);
} else {
 this.setPositionOnRightOfCursor(cursor);
}
```

Reply:
wdyt?

Reply:
You are right, the error is present. And for me it's also less clear.

Putting this problem aside, maybe 2 extra methods will do it more clear or even wrapping this 2 newly created methods by one, more general. ie. `setHorizontalPosition`? WDYT?

Reply:
Oh yeah, I was analyzing this approach, and finally, I figured it was low profit. As you point a problem here, I probably was wrong. 

Ok then, I'll extract methods here.

Reply:
done a176e8d